sdk/translation/azure-ai-translation-text/samples/README.md

```diff
@@ -24,36 +24,103 @@ Translator Service is a cloud-based neural machine translation service that is p
 
 See the [README][README] of the Text Translator client library for more information, including useful links and instructions.
 
-## Common scenarios samples
+# Create Client
 
+Text Translation service is using two types of endpoints - Global and Custom. You can find more information in the [v3 Translator reference][TranslatorReference].
 
-# Create Client
+## Global Endpoint
 
-For some of these operations you can create a new `TextTranslationClient` without any authentication. You will only need your endpoint:
+When using Text Translation service with global endpoint, the endpoint doesn't need to be provided when creating `TextTranslationClient`. When `endpoint` is
+not provided, global `api.cognitive.microsofttranslator.com` is used. For [sovereign clouds][SovereignClouds], the endpoint is always required.
 
-<!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_endpoint -->
+### Using the Subscription Key
+
+When using `cognitive services key` and `region` with global endpoint, you can create `TextTranslationClient`:
+
+<!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_credential -->
 
 ```python
-text_translator = TextTranslationClient(endpoint=endpoint)
+credential = AzureKeyCredential(apikey)
+text_translator = TextTranslationClient(credential=credential, region=region)
 ```
 
 <!-- END SNIPPET -->
 
-The values of the `endpoint` variable can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+### Using the Cognitive Services Token
 
-For other samples an overloaded constructor is provided that uses a TextTranslationCredential.  In addition to `endpoint`, this function requires configuring an `apikey` and `region` to create the credential.  The values of the `endpoint`, `apiKey` and `region` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+For the Cognitive Services Token authentication, there is currently no implementation provided in the SDK. You can implement the solution
+according to the [Token Authentication documentation][TranslatorReference]. When the token is created, you can create the `TextTranslationClient`:
```

Comment:
Sorry, I'm still not understanding this statement. So we don't provide a credential type for cognitive services token in the library and suggest that users create their own credential. And this will work with the authentication policies we have in the code?

Reply:
Yes, users need to implement this on their own. I am not aware that there is any implementation for the cognitive services tokens. I am not sure if the team behind that token is planning to implement it or not.